taskgraph/Task.py

```diff
@@ -1467,3 +1373,69 @@ def _normalize_path(path):
             "failed to abspath %s so returning normalized path instead")
         abs_path = norm_path
     return os.path.normcase(abs_path)
+
+
+@retrying.retry(wait_exponential_multiplier=1000, wait_exponential_max=5000)
+def _execute_sqlite(
+        sqlite_command, database_path, argument_list=None,
+        mode='read_only', execute='execute', fetch=None):
+    """Execute SQLite command and attempt retries on a failure.
+
+    Parameters:
+        sqlite_command (str): a well formatted SQLite command.
+        database_path (str): path to the SQLite database to operate on.
+        mode (str): must be either 'read_only' or 'modify'.

```

Comment:
Looks like `argument_list` is missing from the parameter description here.

Reply:
Documenting in 0cd75ea